src/main/java/com/bina/varsim/util/VCFparser.java

```diff
@@ -347,7 +347,13 @@ else if (index == 9) { // Output format
                 }
             }
         }
-        Alt[] alts = string2Alt(ALT);
+        Alt[] alts = null;
+        try {
+            alts = string2Alt(ALT);
+        } catch (IllegalArgumentException e) {
+            log.warn(e.getMessage());
```

Comment:
Would be good to print out the offending VCF line

Reply:
good point, just did. Thanks.